src/languages/translations.js

```diff
@@ -0,0 +1,9 @@
+import en from './en';
+import es from './es';
+import esES from './es-ES';
```

Comment:
Can we not name this `es-ES` instead of `esES` ?

Reply:
In JS, variable/module/function names can't contain `-`. Maybe, we can use `_` here instead.